src/com/apeshko/javacore/task_01/Application.java

```diff
@@ -0,0 +1,244 @@
+package com.apeshko.javacore.task_01;
+
+import com.apeshko.javacore.task_01.model.*;
+import com.apeshko.javacore.task_01.model.exception.NotEnoughDistanceException;
+import com.apeshko.javacore.task_01.sevice.AirCompanySearchService;
+import com.apeshko.javacore.task_01.sevice.AirCompanyService;
+import com.apeshko.javacore.task_01.sevice.impl.AirCompanySearchServiceImpl;
+import com.apeshko.javacore.task_01.sevice.impl.AirCompanyServiceImpl;
+import com.apeshko.javacore.task_01.sevice.impl.FlightServiceImpl;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+
+public class Application {
+    private final static List<AirCompany> airCompanies = Arrays.asList(
+            new AirCompany(
+                "WizzAir",
+                Arrays.asList(
+                    new Airplane.Builder()
+                            .withName("Sea")
+                            .withCapacity(120)
+                            .withType(Airplane.Type.AIRBUS)
```

Comment:
if this enum reachable not only from Airplane class, move it to separate class, AirplaneType.AIRBUS
each public class should be in separate class file, enum is a class as well. Reasonable embedding - if enum accessible only from the class he embedded into.